src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -873,10 +874,32 @@ protected virtual void Dispose(bool disposing)
 
         public void Dispose()
         {
+            StopWorkerProcessAsync();
             _disposing = true;
             Dispose(true);
         }
 
+        public void StopWorkerProcessAsync()
```

Comment:
Should this be called `StopWorkerProcess`? I don't think we're awaiting anything or returning a task for it to be async